### ISSUE TITLE
Form Widget (Pro) - <optgroup> tag

### ISSUE THREAD

**Issue by mepacsk** [elementor/elementor#10401]
After a good response from Elementor support please add a new feature to Form Widget (select ) to support optional groups <optgroup> tag.
In the dropdown menu to support groups inside. Group can`t be selected. 
For example: 
--- Group 1 ---
Select value 1
Select value 2
-- Group 2 ---
Select value 1
Select value 2

**Comment by pocketsocialau** [comment]
This would be a very useful feature.

**Comment by fippy** [comment]
+1

**Comment by longbkit** [comment]
👍 This would be helpful.

**Comment by azadrpi** [comment]
I'm also looking for this feature!

**Comment by npanti** [comment]
+1

**Comment by Xavvi** [comment]
+1

**Comment by ravenswoodmedia** [comment]
+1

**Comment by Mr-montasir** [comment]
Yeah, can't wait to get this feature!

**Comment by javacious** [comment]
Yes, please add optgroup! Forms are incredibly powerful and drive actual business objectives. If Elementor could focus more on their form product than say... AI which has been practically useless, a lot of us business operators would be very pleased.

I'll add that the multi select field needs a major upgrade. The modern standard allows text searching and tag-like UI.

Please Elementor, work on your forms! :)

**Comment by caspervoogt** [comment]
I would love this
